contrib/gocql/gocql/gocql.go

```diff
@@ -163,3 +176,92 @@ func (tIter *Iter) Close() error {
 	tIter.span.Finish()
 	return err
 }
+
+// Scanner returns a row Scanner which provides an interface to scan rows in a
+// manner which is similar to database/sql. The Iter should NOT be used again after
+// calling this method.
+func (tIter *Iter) Scanner() gocql.Scanner {
+	return &Scanner{
+		Scanner: tIter.Iter.Scanner(),
+		Iter:    tIter,
+	}
+}
+
+// Err calls the wrapped Scanner.Err, releasing both Scanner and Iter resources.
+func (s *Scanner) Err() error {
+	if err := s.Iter.Close(); err != nil {
+		s.Scanner.Err()
```

Comment:
Why is this call here? It's a bit odd and I wonder if it warrants a comment explaining the reason.

Reply:
I realized this implementation was an artifact of how I initially wrapped the package. The most recent commit mirrors the Iter.Close function directly, which is cleaner.